libs/server/src/lib/utils/read-projects.ts

```diff
@@ -20,7 +20,7 @@ export function readProjects(json: any): Project[] {
         name: key,
         root: value.root,
         projectType: value.projectType,
```

Comment:
Sorry if this is a dumb question 😬  Do you know what calls this `readProjects` function? Nx 11 supports targets too, and I added support for reading targets as architect in #996. However I skipped over this because I didn't see it used anywhere and thought it was dead code.